src/dialects/abstract/query-interface.js

```diff
@@ -353,7 +353,7 @@ export class QueryInterface {
       type: QueryTypes.SHOWTABLES,
     };
 
-    const showTablesSql = this.queryGenerator.showTablesQuery(this.sequelize.config.database);
+    const showTablesSql = this.queryGenerator.showTablesQuery(options.schema || this.sequelize.config.database);
```

Comment:
Not related to your PR (and I don't think you should fix it here), but this method receiving `this.sequelize.config.database` sounds like a legacy bad design to me.